lib/localio/localizable_writer.rb

```diff
@@ -1,5 +1,6 @@
 require 'localio/writers/android_writer'
 require 'localio/writers/ios_writer'
+require 'localio/writers/oeg_writer'
```

Comment:
Since this could be resuable by the other front-end apps, I might call it something more generic. Perhaps javascript_writer?